### PR TITLE
PLT-3328 Added Title for Direct Message Desktop Notifications

### PR DESCRIPTION
**Pull request by DavidLu1997**
Previously, direct message notifications did not have a title on desktop. Now the title is `Direct Message`.

### CODE REVIEW

File: webapp/stores/notification_store.jsx [hunk 1]
```diff
@@ -63,7 +63,9 @@ class NotificationStoreClass extends EventEmitter {
             }
 
             let title = Utils.localizeMessage('channel_loader.posted', 'Posted');
-            if (channel) {
+            if (channel.type === Constants.DM_CHANNEL) {
```
Comment (crspeller) [inline webapp/stores/notification_store.jsx]: We are checking `if(channel)` below but `channel.type === Constants.DM_CHANNEL` implies `channel`.  Please fix the logic here.

**Comment by DavidLu1997** [comment]
@crspeller logic fixed

**Comment by crspeller** [comment]
+1

**Comment by enahum** [comment]
+1

**Pull request merged**